pkg/deploy/deploy.go

```diff
@@ -221,11 +221,9 @@ func (c *componentDeployer) deploy(ctx context.Context) error {
 
 		for _, root := range roots {
 			node := root.(graph.ConfigNode)
-			ctx := context.WithValue(ctx, log.CtxKeyCoord{}, node.Config.Coordinate)
-
 			go func(ctx context.Context, node graph.ConfigNode) {
 				errChan <- c.deployNode(ctx, node)
-			}(ctx, node)
+			}(context.WithValue(ctx, log.CtxKeyCoord{}, node.Config.Coordinate), node)
```

Comment:
Wouldn't it be more readable if we kept the var on L224?

Reply:
yes, maybe. just did it because I don't like shadowed variables so much.  I can change it back if you want tho